src/App.js

```diff
@@ -49,17 +49,30 @@ class App extends Component {
         this.setState({
           isDisabledBtn: false,
           web3Config
-        });
-      })
-      .catch(error => {
-        if (error.msg) {
-          this.setState({ isDisabledBtn: true });
-          swal({
-            icon: "warning",
-            title: "Warning",
-            content: error.node
+        })
+          .then(async config => {
```

Comment:
@vbaranov Here was an incorrect merge - the DApp showed the next error:

![image](https://user-images.githubusercontent.com/33550681/47151320-0ef24380-d2e2-11e8-98f9-3de130f8b32e.png)

I've fixed that in https://github.com/poanetwork/poa-dapps-keys-generation/pull/85/commits/5a9d896c321d8745dc5d94640dc3f87f5a94e001